VirtoCommerce.JavaScriptShoppingCart.Web/Scripts/cart/cart.js

```diff
@@ -370,7 +370,8 @@ cartModule.controller('virtoCommerce.cartModule.checkoutController', ['$scope',
 		$uibModalInstance.dismiss('cancel');
     };
 
-    $scope.$on('orderCreated', function (event, data) {
+    $scope.$on('userLoggedIn', function (event, data) {
+		$scope.cart.reloadCart();

```

Comment:
Just note: In my opinion, there are no reasons to name the method 'Reload Cart' for an object that is already has been named 'cart'. We should get rid of useless adjectives/nouns names. The code will be shorter at different levels ...

Reply:
https://github.com/VirtoCommerce/vc-module-javascript-shoppingcart/pull/26/files#diff-68a16e6bfdda2b0bbe097005827c61e9R419

for what we have here "todo"? It just comments or we should do something here?
if this a just information I think it would be better to remove "todo" and not seed here misunderstanding for others

Reply:
https://github.com/VirtoCommerce/vc-module-javascript-shoppingcart/pull/26/files#diff-91b33df1a96b77fff8302b1f82d8696dR33
would be better if we won't use magic numbers anywhere, even if these numbers are well-known HTTP codes

Reply:
https://github.com/VirtoCommerce/vc-module-javascript-shoppingcart/pull/26/files#diff-91b33df1a96b77fff8302b1f82d8696dL29
why we use here 'x'? is this math operations here?) It would be much better to use the clear name, eg: 'request' or something like it

Reply:
fixed